tests/Tests/505__must_gather/test-must-gather-logs.robot

```diff
@@ -0,0 +1,33 @@
+*** Settings ***
+Library   Process
+Library   OperatingSystem
+
+*** Test Cases ***
+Get must-gather logs
+      ${output}    Run process    tests/Tests/505__must_gather/get-must-gather-logs.sh     shell=yes
+      Should Not Contain    ${output.stdout}    FAIL
+      ${must-gather-dir}=  Run     ls -d must-gather.local.*
+      ${namespaces-log-dir}=      Run     ls -d ${must-gather-dir}/quay-io-modh-must-gather-sha256-*/namespaces
+      Set Suite Variable      ${must-gather-dir}
+      Set Suite Variable      ${namespaces-log-dir}
+
+      Directory Should Exist    ${must-gather-dir}
+      Directory Should Not Be Empty   ${must-gather-dir}
+
+Verify logs for redhat-ods-applications
+      Directory Should Exist    ${namespaces-log-dir}/redhat-ods-applications
```

Comment:
Just a thought. Would it be good if we check for specific files exist inside the directory along with directory verification?

Reply:
Updated to recursively search for `*.log` files